refs.h

```diff
@@ -544,6 +544,11 @@ int refs_for_each_reflog(struct ref_store *refs, each_reflog_fn fn, void *cb_dat
  */
 int check_refname_format(const char *refname, int flags);
 
+/*
+  * Return 0 iff all refs in filesystem are consistent.
+*/
+int refs_check_refs(struct ref_store *refs);
```

Comment:
We do not necessarily only check references, but also reflogs and potentially other data structures that the ref store carries. So I would make this a bit more generic and call it `refs_fsck()`, and also adjust the callback function to be namd `fsck` instead of `fsck_refs`.